scipy/stats/tests/test_stats.py

```diff
@@ -586,20 +586,14 @@ def test_kendalltau():
     # empty arrays provided as input
     assert_equal(stats.kendalltau([], []), (np.nan, np.nan))
 
-    # check two different sort methods
-    with warnings.catch_warnings():
-        warnings.simplefilter('ignore', UserWarning)
-        assert_approx_equal(stats.kendalltau(x1, x2, initial_lexsort=False)[1],
-                            stats.kendalltau(x1, x2, initial_lexsort=True)[1])
```

Comment:
initial_lexsort is a fake variable. It has no effect. It is deprecated in the documentation. The test is entirely useless.